templates/_layout.html

```diff
@@ -7,7 +7,9 @@
     <title>{% block title %}{% endblock %}</title>
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet"
         integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
-    {% block styles %}{% endblock %}
+    {% block styles %}
+        <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" rel="stylesheet">
```

Comment:
because this link is inside the styles block, it gets overwitten whenever a different html file defines its own styles. so it should be just above or below the block if you want it to apply to every html page